docs/upgrade-notes/upgrade-rds.md

```diff
@@ -0,0 +1,246 @@
+---
+id: upgrade-rds
+title: Upgrade to RDS release
+hide_title: false
+---
+
+## Background
+
+This release of Cumulus (x.x.x) integrates with RDS and creates a new PostgreSQL database for archiving Cumulus data (e.g. granules, files, executions).
```

Comment:
Is it safe to update `x.x.x` to `9.0.0` now?

Reply:
I would say so, yes.